gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/utils/InlineOrganizationProviderConfiguration.java

```diff
@@ -92,6 +92,7 @@ public IdentityProvider buildIdentityProvider() {
         provider.setExternal(false);
         provider.setType("inline-am-idp");// use the inline provider implementation as InMemory provider
         provider.setName(getName());
+        provider.setSystem(true);
```

Comment:
We don't want to mark the inline identity provider created for the Organization as System. (only default idp created for the domain)